pkg/piperutils/FileUtils.go

```diff
@@ -47,3 +66,23 @@ func Copy(src, dst string) (int64, error) {
 	nBytes, err := io.Copy(destination, source)
 	return nBytes, err
 }
+
+// Copy ...
+func Copy(src, dst string) (int64, error) {
+	return Files{}.Copy(src, dst)
+}
+
+//FileRead ...
+func (f Files) FileRead(path string) ([]byte, error) {
+	return ioutil.ReadFile(path)
+}
+
+// FileWrite ...
+func (f Files) FileWrite(path string, content []byte, perm os.FileMode) error {
+	return ioutil.WriteFile(path, content, perm)
+}
+
+// MkdirAll ...
+func (f Files) MkdirAll(path string, perm os.FileMode) error {
+	return os.MkdirAll(path, perm)
```

Comment:
Do I understand correctly, that the purpose of this all is to be able to mock the file related functionality?

Reply:
Yes, purpose is to intercept file system access for the tests.